virttest/utils_conn.py

```diff
@@ -1447,7 +1453,7 @@ def client_setup(self):
 
 
 def build_client_key(tmp_dir, client_cn="TLSClient", certtool="certtool",
-                     credential_dict=None):
+                     credential_dict=None, ca_cakey_path=None):
```

Comment:
As ca_cakey_path is associated with credentials, would you please add ca_cakey_path into credential_dict?

Reply:
DONE, also updated build_CA() and build_server_key()